ten-simple-rules-dockerfiles.Rmd

```diff
@@ -65,14 +65,14 @@ Computing infrastructure has advanced to the point where not only can we share d
 code that processes these data.
 This sharing of code files is enabled by collaboration platforms such as [GitHub](https://github.com) or [GitLab](https://gitlab.com) and has become quite common.
 The sharing of the computing environment, on the other hand, is enabled by containerization, which allows for documenting and sharing entire workflows in a comprehensive way.
-This sharing of computational assets is paramount to increase reproducibility of computational research.
+This sharing of computational assets is paramount to increasing the reproducibility of computational research.
 While "papers" based on the traditional journal article format can share high level details of research, computational research is oftentimes way too complicated to be disseminated alongside in this format [@marwick_how_2015]. 
 In that the actual contribution to knowledge includes the full computing environment that produced a result [@donoho_invitation_2010], containerization is needed.
 
 Containerization plays an important role in providing instructions for packaging the building blocks of computer-based research (i.e. code, data, documentation, and the computing environment).
-While a container cannot package all of these blocks, it represents a human- _and_ machine-readable recipe to capture the computing environment and interact with data.
+Containers are built from plain text files which represent a human- _and_ machine-readable recipe for creating the computing environment and interact with data.
 By providing this recipe, authors of scientific articles greatly improve their work's level of documentation, transparency, and reusability.
-Such practice is one important part of common practices for scientific computing [@wilson_best_2014; @wilson_good_2017], with the result that it is much more likely both the author and others are able to reproduce and extend an analysis workflow.
+This is an important and common practice in scientific computing [@wilson_best_2014; @wilson_good_2017], with the result that it is much more likely that both the author and others are able to reproduce and extend an analysis workflow.
 The containers built from these recipes are portable encapsulated snapshots of a specific computing environment.
```

Comment:
I like your change here - can we take it a bit further and instead of saying "This" refer directly to the topic at hand?

Reply:
Ok, I've elaborated on this now.